HousingRepairsOnlineApi/Domain/Repair.cs

```diff
@@ -6,6 +6,7 @@ public class Repair
     {
         [JsonProperty("id")]
         public string Id { get; set; }
+        public string RepairType { get; set; } /// Communal, Tenant, Leasehold
```

Comment:
This comment about the potential values could become a disconnected from acceptable values and then are a source of disinformation. It would be best to remove the comment and rely on other documentation.